ruby2.7/run/Dockerfile

```diff
@@ -12,9 +7,9 @@ ENV PATH=/var/lang/bin:$PATH \
     LD_LIBRARY_PATH=/var/lang/lib:$LD_LIBRARY_PATH \
     AWS_EXECUTION_ENV=AWS_Lambda_ruby2.7
 
-COPY --from=0 /opt/* /var/
+ADD https://lambci.s3.amazonaws.com/fs/ruby2.7.tgz /var
```

Comment:
This just copies `ruby2.7.tgz` to `/var/`

> Resources from remote URLs are not decompressed

https://docs.docker.com/engine/reference/builder/#add

Reply:
sorry should not be part of this PR